### DISCUSSION TITLE
markdown syntax

### DISCUSSION THREAD

**Discussion by biglonglong** [chrede88/qubt#198] [Ideas]
How about adding some markdown syntax?
For example formula blocks using $ references, code blocks using ` references

**Comment by chrede88** [comment]
Markdown support is build-in to Hugo. See this blog post (from the example site): https://chrede88.github.io/qubt/blog/post1/

Hugo uses [Goldmark](https://github.com/yuin/goldmark/) to render markdown.
Latex rendering is currently not supported by the theme, but I can add it if it's important to you. See the Hugo docs for more info: https://gohugo.io/content-management/mathematics/#overview

Can you point me to where markdown rendering doesn't work?

**Comment by chrede88** [comment]
I've added Latex rendering to `v2.0.0` of the theme. Please let me know if you have other issues with markdown rendering.

**Comment by biglonglong** [comment]
Thank you very much for your work! It was enough for me to write some markdown files.
But I tried to test all markdown syntaxes and there are two syntaxes that don't support rendering, it's up to you whether to add these renderings or not.
The first one is the extra quotes when using &gt; to quote a piece of content and the second one is that it doesn't support ` to quote short code content.
You can add the following to your markdown file to discover this rendering problem:
&gt; This is a quote bar


`demo`







------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "chrede88/qubt"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年12月7日(星期六) 晚上11:33
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [chrede88/qubt] markdown syntax (Discussion #198)





 
I've added Latex rendering to v2.0.0 of the theme. Please let me know if you have other issues with markdown rendering.

**Reply by chrede88** [reply]
I got the inline code block to render correctly. They're now rendered without the extra single quotes:

old: `'inlineCodeBlock'`
new: `inlineCodeBlock`

I don't really understand what you want me to do about this: `&gt;`?
It renders to a `>` in standard markdown (also here on Github).

**Comment by chrede88** [comment]
I'm closing this, as the inline code block now renders correctly (v2.0.1).
